src/game.zig

```diff
@@ -63,55 +42,55 @@ pub const collision = struct {
 		const f2 = triangle[0] - triangle[2];
 
 		// Test axis a00
-		const a00 = Vec3d{0, -f0[Z], f0[Y]};
+		const a00 = Vec3d{ 0, -f0[Z], f0[Y] };
```

Comment:
Oh no, auto-formatting strikes again :​(
Please all the unnecessary formatting changes in this file.

Reply:
I think something got messed up and formatted while I was merging. I've reverted and re-merged and it should be fixed now.